schema/float_test.go

```diff
@@ -7,6 +7,27 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestFloatQueryValidator(t *testing.T) {
+	s, err := Float{}.ValidateQuery(1.2)
+	assert.NoError(t, err)
+	assert.Equal(t, 1.2, s)
+	s, err = Float{}.ValidateQuery(1)
+	assert.EqualError(t, err, "not a float")
+	assert.Nil(t, s)
+	s, err = Float{}.ValidateQuery("1.2")
+	assert.EqualError(t, err, "not a float")
+	assert.Nil(t, s)
+	s, err = Float{Boundaries: &Boundaries{Min: 0, Max: 2}}.ValidateQuery(3.1)
+	assert.NoError(t, err)
+	assert.Equal(t, 3.1, s)
+	s, err = Float{Boundaries: &Boundaries{Min: 0, Max: 2}}.ValidateQuery(1.1)
+	assert.NoError(t, err)
+	assert.Equal(t, 1.1, s)
+	s, err = Float{Boundaries: &Boundaries{Min: 2, Max: 10}}.ValidateQuery(1.1)
+	assert.NoError(t, err)
+	assert.Equal(t, 1.1, s)
```

Comment:
What about the rest of the tests in the test suit? Wouldn't be awkward to have part of the tests `assert` free, and other part full of asserts? I guess you want me to redo just new tests I have added.

Reply:
If you redo your new tests, that's more than good enough for me. It will be a bit acward to have two different styles within the same file -- could perhaps be fixed by simply putting the new tests in a new file.

- <name>_test.go <- new tests
- <name>_assert_test.go <- rename file with old tests...

Reply:
Okay, I will redo my tests only for now. Lets leave separating the test files for another PR. Depending on how much work it is, I might just redo the old tests as well.